systemd-boot/install

```diff
@@ -8,11 +8,15 @@
 #
 
 err=0
-if [ -x /usr/bin/bootctl ] ; then
-  ( set -x ; bootctl --make-machine-id-directory=yes install ) || err=1
+if [ -x /usr/bin/sdbootutil ] ; then
+  ( set -x ; sdbootutil install ) || err=1
 else
-  echo "bootctl: command not found"
-  err=1
+  if [ -x /usr/bin/bootctl ] ; then
```

Comment:
I am a bit torn here. For openSUSE this can break the system, as shim.efi will be dropped.

But I guess that makes more sense than the links one. So OK for me.